knative-operator/pkg/controller/knativeserving/knativeserving_controller_test.go

```diff
@@ -119,7 +119,6 @@ func init() {
 	os.Setenv("OPERATOR_NAME", "TEST_OPERATOR")
 	os.Setenv("KOURIER_MANIFEST_PATH", "kourier/testdata/kourier-latest.yaml")
 	os.Setenv(dashboard.ServingDashboardPathEnvVar, "../dashboard/testdata/grafana-dash-knative.yaml")
-	os.Setenv("CONSOLECLIDOWNLOAD_MANIFEST_PATH", "consoleclidownload/testdata/console_cli_download_kn_resources.yaml")
```

Comment:
This was removed before but added back by accident(?) in https://github.com/openshift-knative/serverless-operator/commit/afea924e5f5322188a68f1b58002858debcbf47b#diff-3d7f08fe0153d53554c2235e86af9aeb830dba35b37b3d70d84bd51e9100855fR117